Source/SourceKittenFramework/String+SourceKitten.swift

```diff
@@ -80,11 +80,17 @@ extension NSString {
     */
     public func byteRangeToNSRange(start start: Int, length: Int) -> NSRange? {
         let string = self as String
-        return string.indexOfByteOffset(start).flatMap { stringStart in
-            return string.indexOfByteOffset(start + length).map { stringEnd in
-                return NSRange(location: stringStart, length: stringEnd - stringStart)
-            }
+        let startUTF8Index = string.utf8.startIndex.advancedBy(start)
+        let endUTF8Index = startUTF8Index.advancedBy(length)
+        
+        guard let startUTF16Index = startUTF8Index.samePositionIn(string.utf16),
```

Comment:
I tested with changes of that and got same performance.
It seems Swift optimizer is clever enough. :smiley: 
But apply them.